concepticondata/conceptlists/Gerardi-2021-244.tsv

```diff
@@ -0,0 +1,245 @@
+Gerardi-2021-244-119	119	LONG	1203	LONG
+Gerardi-2021-244-120	120	LOOK_AT	1819	LOOK
+Gerardi-2021-244-121	121	LOOSE_oneS	423	GET LOST
+Gerardi-2021-244-122	122	LOUSE	1392	LOUSE
+Gerardi-2021-244-123	123	MAN	1554	MAN
+Gerardi-2021-244-124	124	MANIOC	927	MANIOC
+Gerardi-2021-244-125	125	MANY	1198	MANY
+Gerardi-2021-244-126	126	MEDICINE	1372	MEDICINE
+Gerardi-2021-244-127	127	MONKEY	1350	MONKEY
+Gerardi-2021-244-128	128	MOON	1313	MOON
+Gerardi-2021-244-129	129	MOUNTAIN	639	MOUNTAIN
+Gerardi-2021-244-130	130	MOSQUITO	1509	MOSQUITO
+Gerardi-2021-244-131	131	MOTHER	1216	MOTHER
+Gerardi-2021-244-132	132	MOTHER_IN_LAW (2)	2256	MOTHER-IN-LAW (OF WOMAN)
+Gerardi-2021-244-133	133	MOUSTACHE	1165	MOUSTACHE
+Gerardi-2021-244-134	134	NAIL/CLAW	2128	CLAW OR NAIL
+Gerardi-2021-244-135	135	NAME	1405	NAME
+Gerardi-2021-244-136	136	NECK	1333	NECK
+Gerardi-2021-244-137	137	NEST	539	NEST
+Gerardi-2021-244-138	138	NEW	1231	NEW
+Gerardi-2021-244-139	139	NIGHT	1233	NIGHT
+Gerardi-2021-244-140	140	NOSE	1221	NOSE
+Gerardi-2021-244-141	141	ONE	1493	ONE
+Gerardi-2021-244-142	142	OTHER_SOME	3863	OTHER OR SOME
+Gerardi-2021-244-143	143	PACA	316	PACA
+Gerardi-2021-244-144	144	PAIN/HURT	3864	PAIN OR HURT
+Gerardi-2021-244-145	145	PARROT	882	PARROT
+Gerardi-2021-244-146	146	PATH	2252	PATH
+Gerardi-2021-244-147	147	PENIS	1222	PENIS
+Gerardi-2021-244-148	148	PERSON_HB_SOMEONE	683	PERSON
+Gerardi-2021-244-149	149	PIERCE	398	PIERCE
+Gerardi-2021-244-150	150	PINEAPPLE	2440	PINEAPPLE
+Gerardi-2021-244-151	151	PLAY	1413	PLAY
+Gerardi-2021-244-152	152	PORRIDGE	3651	MUSH (FOOD)
+Gerardi-2021-244-153	153	PUMPKIN	864	PUMPKIN
+Gerardi-2021-244-154	154	PUT	998	PUT
+Gerardi-2021-244-155	155	RAIN	658	RAIN (PRECIPITATION)
+Gerardi-2021-244-156	156	RAT	1490	RAT
+Gerardi-2021-244-157	157	RAW	1959	RAW
+Gerardi-2021-244-158	158	RED	156	RED
+Gerardi-2021-244-159	159	RETURN	581	COME BACK
+Gerardi-2021-244-160	160	RIPE	178	RIPE
+Gerardi-2021-244-161	161	RIVER	666	RIVER
+Gerardi-2021-244-162	162	ROOT	670	ROOT
+Gerardi-2021-244-163	163	ROPE	1218	ROPE
+Gerardi-2021-244-164	164	ROUND	1395	ROUND
+Gerardi-2021-244-165	165	RUN	1519	RUN
+Gerardi-2021-244-166	166	SALIVA	1503	SALIVA
+Gerardi-2021-244-167	167	SALT	1274	SALT
+Gerardi-2021-244-168	168	SAND	671	SAND
+Gerardi-2021-244-169	169	SCARE	252	FRIGHTEN
+Gerardi-2021-244-170	170	SEE	1409	SEE
+Gerardi-2021-244-171	171	SEED	714	SEED
+Gerardi-2021-244-172	172	SEEK	1468	LOOK FOR
+Gerardi-2021-244-173	173	SEND	1127	SEND
+Gerardi-2021-244-174	174	SHARPEN	251	SHARPEN (SOMETHING)
+Gerardi-2021-244-175	175	SHELL/BARK	3865	SHELL OR BARK
+Gerardi-2021-244-176	176	SICK/ILL	1847	SICK
+Gerardi-2021-244-177	177	SISTER	1263	SISTER
+Gerardi-2021-244-178	178	SIT	1416	SIT
+Gerardi-2021-244-179	179	SLEEP	1585	SLEEP
+Gerardi-2021-244-180	180	SMALL	1246	SMALL
+Gerardi-2021-244-181	181	SMOKE	778	SMOKE (EXHAUST)
+Gerardi-2021-244-182	182	SNAKE	730	SNAKE
+Gerardi-2021-244-183	183	SNORE	1983	SNORE
+Gerardi-2021-244-184	184	SON	1620	SON
+Gerardi-2021-244-185	185	SOUR/ACID		
```

Comment:
1906 sour (even if it is something that can also mean ACID, I think sour is enough).